joomla/joomla-cms
# [3.9.24] #Bug #32272

Pull Request for Issue #32272 .

### Summary of Changes

Fixed  send data to the user in Privacy component

### Testing Instructions

Follow instructions at #32272 , confirm the issue
Apply patch, confirm the issue is fixed.
